widgetastic_manageiq/__init__.py

```diff
@@ -1340,7 +1340,7 @@ def set_items_per_page(self, value):
 
         items_text = VersionPick({
             Version.lowest(): '{} items'.format(value),
-            '5.8.2': str(value),
+            '5.7.4': str(value),
```

Comment:
What about `5.8.0` and `5.8.1` here - shouldn't it be `'{} items`? @izapolsk

Reply:
@jkrocil, I assume we won't test older 5.8 versions. However, I can use if statement there if you believe those versions can be tested.

Reply:
Yeh, I'd say it doesn't really matter too. I mainly asked because you did take `5.8.0` and `5.8.1` into consideration above.